src/util/fields.py

```diff
@@ -833,6 +833,7 @@ def _update_field_usage_multi(cr, models, old, new, domain_adapter=None, skip_in
 
     p = {
         "old": r"\y%s\y" % (re.escape(old),),
+        "old_pattern": r"\.\y%s\y|\"\y%s\y\"|\'\y%s\y\'" % (re.escape(old), re.escape(old), re.escape(old)),
```

Comment:
I think it could be simpler.
```suggestion
        "old_pattern": r"[\."']%s\y" % (re.escape(old), re.escape(old), re.escape(old)),
```

Reply:
What about dict keys "<pattern> something something"?

Reply:
> What about dict keys ">pattern< something something"?

`r"([\."'])%s\1"` ?

Reply:
actually only for the ' or " but it won't be more readable than what you already have so...

Reply:
Still the `\y` are not needed, so I'd recommend to put the cases explicitly:
`r"""\.%s\y|"%s"|'%s'"""`
In principle you could pass a mapping to `%` to avoid escaping the same thing three times.

Reply:
I've changed it to something more appealing